### PR TITLE
Bundle syncthing 1.13.0 universal binary

### PR DESCRIPTION
**Pull request by mstansberry**
* Updated to use https://github.com/syncthing/syncthing/releases/tag/v1.13.0 `universal` binary

### CODE REVIEW

File: syncthing/Info.plist [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.12.1-1</string>
+	<string>1.13.0-1</string>
 	<key>CFBundleVersion</key>
 	<string>101200101</string>
```
Comment (xor-gate) [inline syncthing/Info.plist]: You need to set the CFBundleVersion to `101300001`

**Review by xor-gate — CHANGES_REQUESTED** [review]

**Comment by xor-gate** [comment]
You could also use the https://github.com/syncthing/syncthing-macos/blob/develop/update-release.py to update the plist and resource fetch script.

File: syncthing/Info.plist (hunk 1 above)
Comment (mstansberry) [reply syncthing/Info.plist]: thank you, update pushed

**Review by xor-gate — APPROVED** [review]

**Comment by xor-gate** [comment]
Thanks @mstansberry !

**Comment by xor-gate** [comment]
Hi @calmh it still seems the app is not notarised with the latest build on develop for some unknown reason (which is a repeat of https://github.com/syncthing/syncthing-macos/pull/142#issuecomment-760803011).

- I downloaded and tested the build from https://build.syncthing.net/viewLog.html?buildId=92192&buildTypeId=SyncthingMacOS_BuildReleaseDmg&tab=artifacts
- Notarise action was also executed https://build.syncthing.net/viewLog.html?buildId=92195&buildTypeId=Release_NotarizeSyncthingMacosMac

Can you help me out?

**Comment by calmh** [comment]
Screenshot 2021-02-02 at 21 26 05

**Comment by xor-gate** [comment]
Thanks @calmh it seems it took some time to propagate.

**Pull request merged**